go.mod

```diff
@@ -164,7 +164,7 @@ require (
 	golang.org/x/crypto v0.31.0 // indirect
 	golang.org/x/exp v0.0.0-20241009180824-f66d83c29e7c // indirect
 	golang.org/x/mod v0.21.0 // indirect
-	golang.org/x/net v0.33.0 // indirect
+	golang.org/x/net v0.30.0 // indirect
```

Comment:
Why is this version being downgraded?

Reply:
I wasn't able to compile successfully without downloading [golang.org/x/net](http://golang.org/x/net)
I doubled check today, and somehow I was able to compile without downgrading... so I dropped the downgrade commit